sakai-evaluation-tool/src/webapp/component-templates/render_text_item.html

```diff
@@ -35,8 +35,10 @@ <h5 class="title">
 				<span rsf:id="itemText" class="label">Item text label.</span>
 
 				<!-- Previous
```

Comment:
I added those 3 new commits, but looks like this block is unchanged. 

Though from what I can see it matches what's in your repo?

https://source.sakaiproject.org/websvn/blame.php?repname=sakai-contrib&path=%2Fuct%2Fevaluations%2Fbranches%2Fvula-10.x%2Ftool%2Fsrc%2Fwebapp%2Fcomponent-templates%2Frender_text_item.html&rev=86865